app/views/admin/applications/show.blade.php

```diff
@@ -117,8 +117,14 @@
                 @endif
               </div>
             @endif
+            @else
```

Comment:
do you mean it could be written in a way that is clearer or are you thinking it is working backwards?